sql/updates/world/master/2024_99_99_99_ragefire_chasm_instance_smart_ai.sql

```diff
@@ -0,0 +1,608 @@
+-- Ragefire Chasm Quests and SAI for Scouts/Rangers and Misc
+UPDATE `creature_template_difficulty` SET `ContentTuningID`=207, `StaticFlags1`=0 WHERE `Entry`=61788; -- SI:7 Ranger
+UPDATE `creature_template_difficulty` SET `ContentTuningID`=207, `StaticFlags1`=0 WHERE `Entry`=61680; -- Kor'kron Scout
```

Comment:
Check current value for StaticFlags1 (probably you're removing a serverside flag) and drop data for Difficulty 1 for every creature entry